src/Workspaces/Core/Portable/Workspace/Solution/VersionStamp.cs

```diff
@@ -168,6 +168,25 @@ public bool Equals(VersionStamp version)
         public static bool operator !=(VersionStamp left, VersionStamp right)
             => !left.Equals(right);
 
+        /// <summary>
+        /// Check whether given persisted version is re-usable. Used by VS for Mac
```

Comment:
Do we have an external access layer for VS for Mac? If so, it might be worth adding a forwarding helper to that layer.

Reply:
No, we don't.